emcc.py

```diff
@@ -3248,7 +3251,17 @@ def modularize():
   with open(final_js, 'w') as f:
     f.write(src)
 
-    # Export using a UMD style export, or ES6 exports if selected
+    # Add ; if src doesn't contain it so we generate valid JS (otherwise acorn fails)
+    if src.rstrip()[-1] != ';':
+      f.write(';')
+
+    # Store the export on the global scope for audio worklets
+    if settings.ENVIRONMENT_MAY_BE_AUDIOWORKLET:
+      f.write('''if (typeof AudioWorkletGlobalScope === 'function')
```

Comment:
It's where all the exports are registered (check just below this) - I think it's because the EXPORT_NAME gets replaced this can't be in `postamble.js`.

And yes, good call on fstrings, but the rest of the module registration code below doesn't use them so I just went with the same style. Will change it.